clients/admin-ui/src/features/privacy-requests/PrivacyRequest.tsx

```diff
@@ -38,8 +39,14 @@ const PrivacyRequest = ({ data: initialData }: PrivacyRequestProps) => {
   const isManualStepsRequired =
     subjectRequest.status === PrivacyRequestStatus.REQUIRES_INPUT;
 
+  // Check if any manual-process integrations exist
+  const { data: manualIntegrations } = useGetManualIntegrationsQuery();
+  const hasManualIntegrations = (manualIntegrations || []).length > 0;
+
+  const showManualSteps = isManualStepsRequired && hasManualIntegrations;
+
   const [activeTabKey, setActiveTabKey] = useState(
-    isManualStepsRequired ? "manual-steps" : "activity",
+    showManualSteps ? "manual-steps" : "activity",
```

Comment:
@lucanovera I did this to prevent the privacy request detail page from automatically switching to the manual steps tab. To my knowledge, that tab is meant for the old "manual webhook" feature.